page/renderer/ComponentUtils.js

```diff
@@ -0,0 +1,67 @@
+export default class ComponentUtils {
+  /**
+   * Initializes the registry used for managing component utils.
+   *
+   * @param {ObjectContainer} oc The application's dependency injector - the
+   *        object container.
+   */
+  constructor(oc) {
+    /**
+     * The application's dependency injector - the object container.
+     *
+     * @type {ObjectContainer}
+     */
+    this._oc = oc;
+
+    /**
+     * Map of registered utilities.
+     *
+     * @type {Object<string, Object>}
+     */
+    this._utilities = {};
+  }
+
+  /**
+   * Registers single utility class or multiple classes in alias->class mapping.
+   *
+   * @param {function(new: Object)|Object<string, function(new: Object)>} componentUtilityClass
+   * @param {string|null} alias
+   * @returns {Object|Object<string, Object>}
+   */
+  register(componentUtilityClass, alias = null) {
```

Comment:
I would use different order of arguments. It will be same as ObjectContainer.bind https://github.com/seznam/IMA.js-core/blob/master/ObjectContainer.js#L123

Reply:
Same as above https://github.com/seznam/IMA.js-core/pull/167#discussion_r316129100